torch/nn/_intrinsic/quantized/modules/conv_relu.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from torch.nn.quantized import Conv2d
+from torch.nn._intrinsic import ConvReLU2d as NNConvReLU2d
+import torch
+
+class ConvReLU2d(Conv2d):
+    r"""
+    A ConvReLU2d module is a fused module of Conv2d and ReLU
+
+    We adopt the same interface as :class:`torch.nn.quantized.Conv2d`.
+
+    Attributes:
+        Same as torch.nn.quantized.Conv2d
+
+    """
+    __FLOAT_MODULE__ = NNConvReLU2d
```

Comment:
it's not there yet for quantized.conv, but I can add this.
also please change to `__FLOAT_MODULE` since that means class attribute

Reply:
Renamed to __FLOAT_MODULE